draft-irtf-cfrg-randomness-improvements.md

```diff
@@ -254,11 +256,17 @@ that generates private values, such as key shares, with:
 G'(n) = HKDF-Expand(HKDF-Extract(H(Sig(sk, tag1)), G(L)), tag2, n)
 ~~~
 
+# Implementation Guidance
+
+Recall that the wrapper defined in {{wrapper}} requires L >= n - L', where L is the Extract output
+length and n is the desired amount of randomness. Some applications may require n to exceed this bound.
+Wrapper implementations SHOULD support this use case by invoking G'(L) multiple times and concatenating
+the results.
 
 # Acknowledgements
 
 We thank Liliya Akhmetzyanova for her deep involvement in the security assessment in {{SecAnalysis}}.
-We thank Martin Thomson and Rich Salz for their careful readings and useful comments.
+We thank John Mattson, Martin Thomson, Rich Salz for their careful readings and useful comments.
```

Comment:
A misprint: it must be "John Mattsson", not "John Mattson"

Reply:
Whoops! Will fix.